backstage/templates/backstage-app-config.yaml

```diff
@@ -22,8 +22,6 @@ data:
   {{- end }}
   APP_CONFIG_backend_baseUrl: {{ .Values.appConfig.backend.baseUrl | quote | quote }}
   APP_CONFIG_backend_cors_origin: {{ .Values.appConfig.backend.cors.origin | quote | quote }}
-  APP_CONFIG_techdocs_storageUrl: {{ .Values.appConfig.techdocs.storageUrl | quote | quote }}
```

Comment:
These are deprecated upstream and I don't think they're needed. I'm open to correction but AFAIK the default uses the discovery api to find the techdocs static/docs endpoint which uses the publisher config to read the files.